mlir/include/mlir/Target/MIOpenCPP.h

```diff
@@ -204,6 +182,41 @@ class PopulateParamsBase {
       } else {
         input2GemmKVectorizable = false;
       }
+    } else if (opType == mlir::miopen::ConvOpType::Conv2DBwdWeightOpType) {
+      // For input tensor
+      // currently, fix that GemmK (NHiWi) is always vectorizable
```

Comment:
@zjing14  want to check if this comment is correct. what's GemmK and GemmN on backward weight?

Reply:
GemmM = k
GemmK = NHW
GemmN = CYX